supported-libraries-schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "This will be populated with actual link once uploaded",
+  "title": "Schema for the supported libraries list JSON",
+  "type": "array",
+  "default": [],
+  "items": {
+    "type": "object",
+    "required": [
+      "artifact",
+      "minimal_version",
+      "metadata_location",
+      "support_level"
+    ],
+    "properties": {
+      "artifact": {
+        "type": "string",
+        "default": "",
+        "title": "The name of the metadata"
```

Comment:
?

Reply:
Maybe additional metadata like a name that make sense to display can be retrieve from Maven Central based on the groupId, artifactId and version?